managed-services/webhosting/how-to/order-dedicated-ip.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: How to order a dedicated IP address for a Web Hosting plan
+  description: This page explains how to order a dedicated IP address for a Web Hosting plan.
+content:
+  h1: How to order a dedicated IP address for a Web Hosting plan
+  paragraph: This page explains how to order a dedicated IP address for a Web Hosting plan
+tags: webhosting
+dates:
+  validation: 2023-12-20
+  posted: 2023-12-20
+categories:
+  - managed-services
+---
+
+In shared Web Hosting, multiple websites share the same server and its resources. Each website typically has its own domain name, but they all use the same IP address for the server. Scaleway provides the option to purchase a dedicated IP address for your website, even within a shared hosting environment. 
+Some potential advantages of having a dedicated IP address for Web Hosting are:
+
+* **Email Reputation:** A dedicated IP address can be beneficial for email reputation. If your shared server's IP address is associated with spam or malicious activities from other users, it could negatively impact your email deliverability. With a dedicated IP, your email reputation is separate from other users on the same server.
```

Comment:
```suggestion
* **Email reputation:** A dedicated IP address can be beneficial for email reputation. If your shared server's IP address is associated with spam or malicious activities from other users, it could negatively impact your email deliverability. With a dedicated IP, your email reputation is separate from other users on the same server.
```